after upgrade from 0.8.2 to 0.9.1, receiving several errors all inside openapi-fetch

does 0.9.1 require any other updated deps?  

```
> shx rm -rf lib/ && tsc

node_modules/openapi-fetch/dist/index.d.ts:83:42 - error TS2314: Generic type 'FetchOptions' requires 1 type argument(s).

83 export type ParseAsResponse<T, O extends FetchOptions> = O extends {
                                            ~~~~~~~~~~~~

node_modules/openapi-fetch/dist/index.d.ts:119:40 - error TS2314: Generic type 'FetchOptions' requires 1 type argument(s).

119 export type FetchResponse<T, O extends FetchOptions> =
                                           ~~~~~~~~~~~~

node_modules/openapi-fetch/dist/index.d.ts:178:36 - error TS2344: Type 'M' does not satisfy the constraint 'HttpMethod'.

178   P extends PathsWithMethod<Paths, M>,
                                       ~

  node_modules/openapi-fetch/dist/index.d.ts:177:44
    177 export type ClientMethod<Paths extends {}, M> = <
                                                   ~
    This type parameter might need an `extends HttpMethod` constraint.

node_modules/openapi-fetch/dist/index.d.ts:179:31 - error TS2344: Type 'Paths[P]' does not satisfy the constraint '{}'.
  Type 'Paths[PathsWithMethod<Paths, M>]' is not assignable to type '{}'.
    Type 'Paths[Paths[keyof Paths] extends { [K in M]: any; } ? keyof Paths : never]' is not assignable to type '{}'.
      Type 'Paths[keyof Paths]' is not assignable to type '{}'.
        Type 'Paths[string] | Paths[number] | Paths[symbol]' is not assignable to type '{}'.
          Type 'Paths[string]' is not assignable to type '{}'.

179   I extends MaybeOptionalInit<Paths[P], M>,
                                  ~~~~~~~~

node_modules/openapi-fetch/dist/index.d.ts:183:28 - error TS2536: Type 'M' cannot be used to index type 'Paths[P]'.

183 ) => Promise<FetchResponse<Paths[P][M], I[0]>>;
```

so, all I did on that was just upgrade openapi-fetch .. 

package.json:
```{
  "name": "grocy-micro-ui",
  "description": "Micro UI for Grocy",
  "version": "0.0.0",
  "homepage": "",
  "private": true,
  "keywords": [
    "feathers"
  ],
  "author": {},
  "contributors": [],
  "bugs": {},
  "engines": {
    "node": ">= 20.10.0"
  },
  "feathers": {
    "language": "ts",
    "packager": "npm",
    "database": "sqlite",
    "framework": "koa",
    "transports": [
      "rest",
      "websockets"
    ],
    "schema": "typebox"
  },
  "directories": {
    "lib": "src",
    "test": "test"
  },
  "files": [
    "lib/client.js",
    "lib/**/*.d.ts",
    "lib/**/*.shared.js"
  ],
  "main": "lib/client",
  "scripts": {
    "build:openapi": "npx openapi-typescript src/grocyApi/grocyApiV4.spec.json -o src/grocyApi/grocyApiV4.d.ts && npm run bundle:client",
    "bundle:client": "npm run compile && npm pack --pack-destination ./public && cd public && npm install grocy-micro-ui-0.0.0.tgz --legacy-peer-deps",
    "compile": "shx rm -rf lib/ && tsc",
    "dev": "cross-env NODE_ENV=development nodemon -x ts-node src/index.ts",
    "start": "cross-env NODE_ENV=production node lib/",
    "mocha": "cross-env NODE_ENV=test mocha test/ --require ts-node/register --recursive --extension .ts --exit",
    "test": "cross-env NODE_ENV=test npm run migrate && npm run mocha",
    "migrate": "knex migrate:latest",
    "migrate:make": "knex migrate:make",
    "upgrade-deps": "npx npm-check-updates -u --doctor"
  },
  "dependencies": {
    "@feathersjs/adapter-commons": "^5.0.22",
    "@feathersjs/authentication": "^5.0.22",
    "@feathersjs/authentication-client": "^5.0.22",
    "@feathersjs/authentication-local": "^5.0.22",
    "@feathersjs/authentication-oauth": "^5.0.22",
    "@feathersjs/configuration": "^5.0.22",
    "@feathersjs/errors": "^5.0.22",
    "@feathersjs/feathers": "^5.0.22",
    "@feathersjs/knex": "^5.0.22",
    "@feathersjs/koa": "^5.0.22",
    "@feathersjs/schema": "^5.0.22",
    "@feathersjs/socketio": "^5.0.22",
    "@feathersjs/transport-commons": "^5.0.22",
    "@feathersjs/typebox": "^5.0.22",
    "cross-env": "^7.0.3",
    "knex": "^3.1.0",
    "openapi-fetch": "^0.8.2",
    "sqlite3": "^5.1.7",
    "winston": "^3.11.0"
  },
  "devDependencies": {
    "@feathersjs/cli": "^5.0.22",
    "@feathersjs/rest-client": "^5.0.22",
    "@types/mocha": "^10.0.6",
    "@types/node": "^20.11.19",
    "axios": "^1.6.7",
    "mocha": "^10.3.0",
    "nodemon": "^3.0.3",
    "openapi-typescript": "^6.7.4",
    "prettier": "^3.2.5",
    "shx": "^0.3.4",
    "ts-node": "^10.9.2",
    "typescript": "^5.3.3"
  }
}```

I see there's a 0.9.2 now, going to give it a look

Looks like it's the same output

```
G:\grocy-micro-ui>npm run compile

> grocy-micro-ui@0.0.0 compile
> shx rm -rf lib/ && tsc

node_modules/openapi-fetch/dist/index.d.ts:83:42 - error TS2314: Generic type 'FetchOptions' requires 1 type argument(s).

83 export type ParseAsResponse<T, O extends FetchOptions> = O extends {
                                            ~~~~~~~~~~~~

node_modules/openapi-fetch/dist/index.d.ts:119:40 - error TS2314: Generic type 'FetchOptions' requires 1 type argument(s).

119 export type FetchResponse<T, O extends FetchOptions> =
                                           ~~~~~~~~~~~~

node_modules/openapi-fetch/dist/index.d.ts:178:36 - error TS2344: Type 'M' does not satisfy the constraint 'HttpMethod'.

178   P extends PathsWithMethod<Paths, M>,
                                       ~

  node_modules/openapi-fetch/dist/index.d.ts:177:44
    177 export type ClientMethod<Paths extends {}, M> = <
                                                   ~
    This type parameter might need an `extends HttpMethod` constraint.

node_modules/openapi-fetch/dist/index.d.ts:179:31 - error TS2344: Type 'Paths[P]' does not satisfy the constraint '{}'.
  Type 'Paths[PathsWithMethod<Paths, M>]' is not assignable to type '{}'.
    Type 'Paths[Paths[keyof Paths] extends { [K in M]: any; } ? keyof Paths : never]' is not assignable to type '{}'.
      Type 'Paths[keyof Paths]' is not assignable to type '{}'.
        Type 'Paths[string] | Paths[number] | Paths[symbol]' is not assignable to type '{}'.
          Type 'Paths[string]' is not assignable to type '{}'.

179   I extends MaybeOptionalInit<Paths[P], M>,
                                  ~~~~~~~~

node_modules/openapi-fetch/dist/index.d.ts:183:28 - error TS2536: Type 'M' cannot be used to index type 'Paths[P]'.

183 ) => Promise<FetchResponse<Paths[P][M], I[0]>>;
                               ~~~~~~~~~~~


Found 5 errors in the same file, starting at: node_modules/openapi-fetch/dist/index.d.ts:83
```

Dropping back to 0.8.2, no errors found
```
G:\grocy-micro-ui>npm run compile

> grocy-micro-ui@0.0.0 compile
> shx rm -rf lib/ && tsc


G:\grocy-micro-ui>
```

--- a/.changeset/unlucky-pumpkins-march.md
+++ b/.changeset/unlucky-pumpkins-march.md
@@ -1,0 +1,5 @@
+---
+"openapi-fetch": patch
+---
+
+Fix type errors

--- a/packages/openapi-fetch/examples/nextjs/package.json
+++ b/packages/openapi-fetch/examples/nextjs/package.json
@@ -12,7 +12,7 @@
     "react-dom": "18.2.0"
   },
   "devDependencies": {
-    "@types/node": "20.11.19",
+    "@types/node": "20.11.24",
     "@types/react": "18.2.20",
     "@types/react-dom": "18.2.7",
     "openapi-typescript": "workspace:^",

--- a/packages/openapi-fetch/package.json
+++ b/packages/openapi-fetch/package.json
@@ -74,7 +74,7 @@
     "openapi-typescript-fetch": "^1.1.3",
     "superagent": "^8.1.2",
     "typescript": "^5.3.3",
-    "vitest": "^1.2.2",
+    "vitest": "^1.3.1",
     "vitest-fetch-mock": "^0.2.2"
   }
 }

--- a/packages/openapi-fetch/src/index.d.ts
+++ b/packages/openapi-fetch/src/index.d.ts
@@ -1,12 +1,13 @@
 import type {
   ErrorResponse,
-  SuccessResponse,
   FilterKeys,
+  HasRequiredKeys,
+  HttpMethod,
   MediaType,
+  OperationRequestBodyContent,
   PathsWithMethod,
   ResponseObjectMap,
-  OperationRequestBodyContent,
-  HasRequiredKeys,
+  SuccessResponse,
 } from "openapi-typescript-helpers";
 
 // Note: though "any" is considered bad practice in general, this library relies
@@ -80,7 +81,7 @@ type BodyType<T = unknown> = {
   stream: Response["body"];
 };
 export type ParseAs = keyof BodyType;
-export type ParseAsResponse<T, O extends FetchOptions> = O extends {
+export type ParseAsResponse<T, O> = O extends {
   parseAs: ParseAs;
 }
   ? BodyType<T>[O["parseAs"]]
@@ -110,13 +111,7 @@ export type RequestBodyOption<T> =
 export type FetchOptions<T> = RequestOptions<T> &
   Omit<RequestInit, "body" | "headers">;
 
-/** This type helper makes the 2nd function param required if params/requestBody are required; otherwise, optional */
-export type MaybeOptionalInit<P extends {}, M extends keyof P> =
-  HasRequiredKeys<FetchOptions<FilterKeys<P, M>>> extends never
-    ? [(FetchOptions<FilterKeys<P, M>> | undefined)?]
-    : [FetchOptions<FilterKeys<P, M>>];
-
-export type FetchResponse<T, O extends FetchOptions> =
+export type FetchResponse<T, O> =
   | {
       data: ParseAsResponse<
         FilterKeys<SuccessResponse<ResponseObjectMap<T>>, MediaType>,
@@ -174,7 +169,19 @@ export interface Middleware {
   onResponse?: typeof onResponse;
 }
 
-export type ClientMethod<Paths extends {}, M> = <
+/** This type helper makes the 2nd function param required if params/requestBody are required; otherwise, optional */
+export type MaybeOptionalInit<
+  P extends Record<HttpMethod, {}>,
+  M extends keyof P,
+> =
+  HasRequiredKeys<FetchOptions<FilterKeys<P, M>>> extends never
+    ? [(FetchOptions<FilterKeys<P, M>> | undefined)?]
+    : [FetchOptions<FilterKeys<P, M>>];
+
+export type ClientMethod<
+  Paths extends Record<string, Record<HttpMethod, {}>>,
+  M extends HttpMethod,
+> = <
   P extends PathsWithMethod<Paths, M>,
   I extends MaybeOptionalInit<Paths[P], M>,
 >(

--- a/packages/openapi-fetch/test/index.test.ts
+++ b/packages/openapi-fetch/test/index.test.ts
@@ -1,4 +1,3 @@
-import { afterEach, beforeAll, describe, expect, it, vi } from "vitest";
 // @ts-expect-error
 import createFetchMock from "vitest-fetch-mock";
 import createClient, {

--- a/packages/openapi-fetch/test/v7-beta.test.ts
+++ b/packages/openapi-fetch/test/v7-beta.test.ts
@@ -1,4 +1,3 @@
-import { afterEach, beforeAll, describe, expect, it, vi } from "vitest";
 // @ts-expect-error
 import createFetchMock from "vitest-fetch-mock";
 import createClient, {

--- a/packages/openapi-fetch/tsconfig.json
+++ b/packages/openapi-fetch/tsconfig.json
@@ -9,11 +9,11 @@
     "moduleResolution": "NodeNext",
     "noUncheckedIndexedAccess": true,
     "outDir": "dist",
-    "skipLibCheck": true,
+    "skipLibCheck": false,
     "strict": true,
     "target": "ESNext",
     "types": ["vitest/globals"]
   },
   "include": ["src", "test"],
-  "exclude": ["node_modules"]
+  "exclude": ["examples", "node_modules"]
 }

--- a/packages/openapi-typescript-helpers/index.d.ts
+++ b/packages/openapi-typescript-helpers/index.d.ts
@@ -58,14 +58,15 @@ export type OperationRequestBodyMediaContent<T> =
     ? FilterKeys<NonNullable<OperationRequestBody<T>>, "content"> | undefined
     : FilterKeys<OperationRequestBody<T>, "content">;
 /** Return first `content` from a Request Object Mapping, allowing any media type */
-export type OperationRequestBodyContent<T> = FilterKeys<
-  OperationRequestBodyMediaContent<T>,
-  MediaType
-> extends never
-  ?
-      | FilterKeys<NonNullable<OperationRequestBodyMediaContent<T>>, MediaType>
-      | undefined
-  : FilterKeys<OperationRequestBodyMediaContent<T>, MediaType>;
+export type OperationRequestBodyContent<T> =
+  FilterKeys<OperationRequestBodyMediaContent<T>, MediaType> extends never
+    ?
+        | FilterKeys<
+            NonNullable<OperationRequestBodyMediaContent<T>>,
+            MediaType
+          >
+        | undefined
+    : FilterKeys<OperationRequestBodyMediaContent<T>, MediaType>;
 /** Return first 2XX response from a Response Object Map */
 export type SuccessResponse<T> = ResponseContent<FilterKeys<T, OkStatus>>;
 /** Return first 5XX or 4XX response (in that order) from a Response Object Map */
@@ -97,4 +98,5 @@ export type FindRequiredKeys<T, K extends keyof T> = K extends unknown
     ? never
     : K
   : K;
+/** Does this object contain required keys? */
 export type HasRequiredKeys<T> = FindRequiredKeys<T, keyof T>;

--- a/packages/openapi-typescript/package.json
+++ b/packages/openapi-typescript/package.json
@@ -62,7 +62,7 @@
     "typescript": "^5.x"
   },
   "dependencies": {
-    "@redocly/openapi-core": "^1.9.0",
+    "@redocly/openapi-core": "^1.10.3",
     "ansi-colors": "^4.1.3",
     "supports-color": "^9.4.0",
     "yargs-parser": "^21.1.1"
@@ -70,13 +70,13 @@
   "devDependencies": {
     "@types/degit": "^2.8.6",
     "@types/js-yaml": "^4.0.9",
-    "@types/node": "^20.11.19",
+    "@types/node": "^20.11.24",
     "degit": "^2.8.4",
     "del-cli": "^5.1.0",
-    "esbuild": "^0.20.0",
+    "esbuild": "^0.20.1",
     "execa": "^7.2.0",
     "typescript": "^5.3.3",
-    "vite-node": "^1.2.2",
-    "vitest": "^1.2.2"
+    "vite-node": "^1.3.1",
+    "vitest": "^1.3.1"
   }
 }

--- a/packages/openapi-typescript/test/index.test.ts
+++ b/packages/openapi-typescript/test/index.test.ts
@@ -1,4 +1,5 @@
 import { fileURLToPath } from "node:url";
+import { beforeAll, describe, expect, test, vi } from "vitest";
 import openapiTS, { astToString } from "../src/index.js";
 import type { OpenAPI3, OpenAPITSOptions } from "../src/types.js";
 import type { TestCase } from "./test-helpers.js";
@@ -692,7 +693,7 @@ export type operations = Record<string, never>;`,
     );
   }
 
-  it("does not mutate original reference", async () => {
+  test("does not mutate original reference", async () => {
     const schema: OpenAPI3 = {
       openapi: "3.1",
       info: { title: "test", version: "1.0" },

--- a/packages/openapi-typescript/test/lib/ts.test.ts
+++ b/packages/openapi-typescript/test/lib/ts.test.ts
@@ -15,7 +15,7 @@ import {
 } from "../../src/lib/ts.js";
 
 describe("addJSDocComment", () => {
-  it("single-line comment", () => {
+  test("single-line comment", () => {
     const property = ts.factory.createPropertySignature(
       undefined,
       "comment",
@@ -30,7 +30,7 @@ describe("addJSDocComment", () => {
 }`);
   });
 
-  it("multi-line comment", () => {
+  test("multi-line comment", () => {
     const property = ts.factory.createPropertySignature(
       undefined,
       "comment",
@@ -57,7 +57,7 @@ describe("addJSDocComment", () => {
 }`);
   });
 
-  it("escapes internal comments", () => {
+  test("escapes internal comments", () => {
     const property = ts.factory.createPropertySignature(
       undefined,
       "comment",
@@ -77,11 +77,11 @@ describe("addJSDocComment", () => {
 });
 
 describe("oapiRef", () => {
-  it("single part", () => {
+  test("single part", () => {
     expect(astToString(oapiRef("#/components")).trim()).toBe(`components`);
   });
 
-  it("multiple parts", () => {
+  test("multiple parts", () => {
     expect(astToString(oapiRef("#/components/schemas/User")).trim()).toBe(
       `components["schemas"]["User"]`,
     );
@@ -89,7 +89,7 @@ describe("oapiRef", () => {
 });
 
 describe("tsEnum", () => {
-  it("string members", () => {
+  test("string members", () => {
     expect(astToString(tsEnum("-my-color-", ["green", "red", "blue"])).trim())
       .toBe(`enum MyColor {
     green = "green",
@@ -98,7 +98,7 @@ describe("tsEnum", () => {
 }`);
   });
 
-  it("name from path", () => {
+  test("name from path", () => {
     expect(
       astToString(
         tsEnum("#/paths/url/get/parameters/query/status", [
@@ -112,7 +112,7 @@ describe("tsEnum", () => {
 }`);
   });
 
-  it("string members with numeric prefix", () => {
+  test("string members with numeric prefix", () => {
     expect(astToString(tsEnum("/my/enum/", ["0a", "1b", "2c"])).trim())
       .toBe(`enum MyEnum {
     Value0a = "0a",
@@ -121,7 +121,7 @@ describe("tsEnum", () => {
 }`);
   });
 
-  it("number members", () => {
+  test("number members", () => {
     expect(astToString(tsEnum(".Error.code.", [100, 101, 102])).trim())
       .toBe(`enum ErrorCode {
     Value100 = 100,
@@ -130,7 +130,7 @@ describe("tsEnum", () => {
 }`);
   });
 
-  it("number members with x-enum-descriptions", () => {
+  test("number members with x-enum-descriptions", () => {
     expect(
       astToString(
         tsEnum(
@@ -153,7 +153,7 @@ describe("tsEnum", () => {
 }`);
   });
 
-  it("x-enum-varnames", () => {
+  test("x-enum-varnames", () => {
     expect(
       astToString(
         tsEnum(
@@ -173,7 +173,7 @@ describe("tsEnum", () => {
 }`);
   });
 
-  it("x-enum-varnames with numeric prefix", () => {
+  test("x-enum-varnames with numeric prefix", () => {
     expect(
       astToString(
         tsEnum(
@@ -189,7 +189,7 @@ describe("tsEnum", () => {
 }`);
   });
 
-  it("partial x-enum-varnames and x-enum-descriptions", () => {
+  test("partial x-enum-varnames and x-enum-descriptions", () => {
     expect(
       astToString(
         tsEnum(
@@ -209,7 +209,7 @@ describe("tsEnum", () => {
 }`);
   });
 
-  it("x-enum-descriptions with x-enum-varnames", () => {
+  test("x-enum-descriptions with x-enum-varnames", () => {
     expect(
       astToString(
         tsEnum(
@@ -237,7 +237,7 @@ describe("tsEnum", () => {
 });
 
 describe("tsPropertyIndex", () => {
-  it("numbers -> number literals", () => {
+  test("numbers -> number literals", () => {
     expect(astToString(tsPropertyIndex(200)).trim()).toBe(`200`);
     expect(astToString(tsPropertyIndex(200.5)).trim()).toBe(`200.5`);
     expect(astToString(tsPropertyIndex(Infinity)).trim()).toBe(`Infinity`);
@@ -245,7 +245,7 @@ describe("tsPropertyIndex", () => {
     expect(astToString(tsPropertyIndex(10e3)).trim()).toBe(`10000`);
   });
 
-  it("valid strings -> identifiers", () => {
+  test("valid strings -> identifiers", () => {
     expect(astToString(tsPropertyIndex("identifier")).trim()).toBe(
       `identifier`,
     );
@@ -257,7 +257,7 @@ describe("tsPropertyIndex", () => {
     expect(astToString(tsPropertyIndex("10e3")).trim()).toBe(`"10e3"`);
   });
 
-  it("invalid strings -> string literals", () => {
+  test("invalid strings -> string literals", () => {
     expect(astToString(tsPropertyIndex("kebab-case")).trim()).toBe(
       `"kebab-case"`,
     );
@@ -273,27 +273,27 @@ describe("tsPropertyIndex", () => {
 });
 
 describe("tsIsPrimitive", () => {
-  it("null", () => {
+  test("null", () => {
     expect(tsIsPrimitive(NULL)).toBe(true);
   });
 
-  it("number", () => {
+  test("number", () => {
     expect(tsIsPrimitive(NUMBER)).toBe(true);
   });
 
-  it("string", () => {
+  test("string", () => {
     expect(tsIsPrimitive(STRING)).toBe(true);
   });
 
-  it("boolean", () => {
+  test("boolean", () => {
     expect(tsIsPrimitive(BOOLEAN)).toBe(true);
   });
 
-  it("array", () => {
+  test("array", () => {
     expect(tsIsPrimitive(ts.factory.createArrayTypeNode(STRING))).toBe(false);
   });
 
-  it("object", () => {
+  test("object", () => {
     expect(
       tsIsPrimitive(
         ts.factory.createTypeLiteralNode([
@@ -310,21 +310,21 @@ describe("tsIsPrimitive", () => {
 });
 
 describe("tsUnion", () => {
-  it("none", () => {
+  test("none", () => {
     expect(astToString(tsUnion([])).trim()).toBe(`never`);
   });
 
-  it("one", () => {
+  test("one", () => {
     expect(astToString(tsUnion([STRING])).trim()).toBe(`string`);
   });
 
-  it("multiple (primitive)", () => {
+  test("multiple (primitive)", () => {
     expect(
       astToString(tsUnion([STRING, STRING, NUMBER, NULL, NUMBER, NULL])).trim(),
     ).toBe(`string | number | null`);
   });
 
-  it("multiple (const)", () => {
+  test("multiple (const)", () => {
     expect(
       astToString(
         tsUnion([NULL, tsLiteral("red"), tsLiteral(42), tsLiteral(false)]),
@@ -332,7 +332,7 @@ describe("tsUnion", () => {
     ).toBe(`null | "red" | 42 | false`);
   });
 
-  it("multiple (object types)", () => {
+  test("multiple (object types)", () => {
     const obj = ts.factory.createTypeLiteralNode([
       ts.factory.createPropertySignature(undefined, "foo", undefined, STRING),
     ]);

--- a/packages/openapi-typescript/test/transform/operation-object.test.ts
+++ b/packages/openapi-typescript/test/transform/operation-object.test.ts
@@ -227,17 +227,32 @@ responses: {
         given: {
           parameters: [
             { in: "path", name: "user_id", schema: { type: "string" } },
-            { in: "path", name: "user_id_deprecated", schema: { type: "string", deprecated: true } },
+            {
+              in: "path",
+              name: "user_id_deprecated",
+              schema: { type: "string", deprecated: true },
+            },
             { in: "query", name: "search", schema: { type: "string" } },
-            { in: "query", name: "search_deprecated", schema: { type: "string", deprecated: true } },
+            {
+              in: "query",
+              name: "search_deprecated",
+              schema: { type: "string", deprecated: true },
+            },
           ],
           responses: {
             "200": {
               description: "OK",
-              content: { "application/json": { schema: { type: "object", properties: { 
-                hello: { type: "string" },
-                hello_deprecated: { type: "string", deprecated: true } 
-              }}}},
+              content: {
+                "application/json": {
+                  schema: {
+                    type: "object",
+                    properties: {
+                      hello: { type: "string" },
+                      hello_deprecated: { type: "string", deprecated: true },
+                    },
+                  },
+                },
+              },
             },
           },
         },

--- a/packages/openapi-typescript/test/yaml.test.ts
+++ b/packages/openapi-typescript/test/yaml.test.ts
@@ -7,7 +7,7 @@ const cwd = os.platform() === "win32" ? fileURLToPath(root) : root; // execa bug
 const cmd = "./bin/cli.js";
 
 describe("YAML features", () => {
-  it("merge", async () => {
+  test("merge", async () => {
     const result = await execa(cmd, ["./test/fixtures/yaml-merge.yaml"], {
       cwd,
     });

--- a/packages/openapi-typescript/tsconfig.json
+++ b/packages/openapi-typescript/tsconfig.json
@@ -2,6 +2,7 @@
   "extends": "../../tsconfig.json",
   "compilerOptions": {
     "lib": ["ESNext", "DOM"],
+    "skipLibCheck": true,
     "sourceRoot": ".",
     "outDir": "dist",
     "types": ["vitest/globals"]

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -7,7 +7,6 @@
     "module": "NodeNext",
     "moduleResolution": "NodeNext",
     "removeComments": true,
-    "skipLibCheck": true,
     "sourceMap": true,
     "strict": true,
     "target": "ESNext",

